beacon-chain/casper/sharding.go

```diff
@@ -1,23 +1,24 @@
 package casper
 
 import (
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/prysmaticlabs/prysm/beacon-chain/params"
-	"github.com/prysmaticlabs/prysm/beacon-chain/types"
 	"github.com/prysmaticlabs/prysm/beacon-chain/utils"
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
 )
 
 // BeaconCommittee structure encompassing a specific shard and validator indices
 // within that shard's committee.
 type BeaconCommittee struct {
-	shardID   int
-	committee []int
+	ShardID   int
+	Committee []int
 }
 
 // ValidatorsByHeightShard splits a shuffled validator list by height and by shard,
 // it ensures there's enough validators per height and per shard, if not, it'll skip
 // some heights and shards.
-func ValidatorsByHeightShard(crystallized *types.CrystallizedState) ([]*BeaconCommittee, error) {
-	indices := ActiveValidatorIndices(crystallized)
+func ValidatorsByHeightShard(seed common.Hash, validators []*pb.ValidatorRecord, dynasty uint64, crosslinkStartShard uint64) ([]*BeaconCommittee, error) {
+	indices := ActiveValidatorIndices(validators, dynasty)
```

Comment:
I think `activeValidators` would be more clear, like the change you made in the casper files.